tools/js-sdk-release-tools/src/utils/version.ts

```diff
@@ -59,7 +59,7 @@ export function bumpPreviewVersion(version: string, usedVersions: string[] | und
 
 export function getNewVersion(stableVersion: string | undefined, usedVersions: string[] | undefined, hasBreakingChange, isStableRelease: boolean): string {
     if (!stableVersion) {
-        logger.error(`Invalid stableVersion ${stableVersion}`);
+        logger.error(`Invalid stableVersion '${stableVersion}'.`);
```

Comment:
Any action to take for the user?

Reply:
Internal error, user should find us in support channel, which is already added in outermost action.